cluster-autoscaler/cloudprovider/aws/README.md

```diff
@@ -215,9 +215,9 @@ This is because the GPU resource does not become available immediately after the
 
 > Note: The minimum version of cluster autoscaler to support MixedInstancePolicy is v1.14.x.
 
-It is possible to use Cluster Autoscaler with a [mixed instances policy](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-autoscaling-autoscalinggroup-mixedinstancespolicy.html), to enable diversification across on-demand and spot instances, of multiple instance types in a single ASG. When using spot instances, this increases the likelihood of successfully launching a spot instance to add the desired capacity to the cluster versus a single instance type, which may be in short supply.
+It is possible to use Cluster Autoscaler with a [mixed instances policy](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-autoscaling-autoscalinggroup-mixedinstancespolicy.html), to enable diversification across on-demand and Spot Instances, of multiple instance types in a single ASG. When using Spot Instances, this increases the likelihood of successfully launching a Spot Instance to add the desired capacity to the cluster versus a single instance type, which may be in short supply.
 
-Note that the instance types should have the same amount of RAM and number of CPU cores, since this is fundamental to CA's scaling calculations. Using mismatched instances types can produce unintended results.
+Note that the instance types should have the same amount of RAM and number of CPU cores, since this is fundamental to CA's scaling calculations. Using mismatched instances types can produce unintended results, See an example below.
```

Comment:
```suggestion
Note that the instance types should have the same amount of RAM and number of CPU cores, since this is fundamental to CA's scaling calculations. Using mismatched instances types can produce unintended results. See an example below.
```